docs/user-guide/how-to.md

```diff
@@ -123,6 +123,53 @@ jobs:
 Read more about [terraform-docs GitHub Action] and its configuration and
 examples.
 
+## Pre-commit Hook
+
+With [`pre-commit`], you can ensure your Terraform module documentation is kept
+up-to-date each time you make a commit.
+
+First, simply create or update a `.pre-commit-hooks.yaml`
+in the root of your Git repo with at least the following content:
```

Comment:
Nit: `s/Git/git/`

Reply:
My understanding is that Git (the project, not the command) should be capitalized here, as it is in the [Git documentation](https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository).

I noticed it's not capitalized it most of the documentation though, so I'm happy to change it if consistency is preferred.

Reply:
That's a very good point! Ignore my comment and keep it as `Git`!